elements/multiple-choice/multiple-choice.js

```diff
@@ -306,12 +313,29 @@ class MultipleChoice extends SchemaBehaviors(PolymerElement) {
       this.$.toast.show();
     }
     // start of data passing, this is a prototype atm
+    let eventData = [""];
+    eventData = {
+      dbType: "xapistatements",
```

Comment:
thinking what if we allow for like two forms of submission. There's pure xapi and then like a simplified form of it which this is driving towards. xapistatements could be the default "database" or something sets default (again, so that we aren't assuming where developers want to send the statements as far as storage location and have it be hard coded).

```js
eventData = {
      type: "xapi",
      activity: "answered",
      objectId: "http://haxcms.psu.edu/haxQuiz",
      name: this.quizName,
      description: "HAX Quiz",
      result: gotRight
    };
```

then all these can be assumed as defaults:
```
resultScoreScaled: 1,
      resultScoreMin: 0,
      resultScoreMax: 100,
      resultScoreRaw: 100,
      resultSuccess: gotRight,
      resultCompletion: true,
      resultResponse: "sample",
      resultDuration: "sample"
```

We also might want to play w/ a paradigm where the event is registered / modified in a global / store kind of concept and that the dispatching of events are more of a "user answered X, it's of type SelfCheck or Quiz" and that those classes are defined elsewhere. This could keep the definitions of how to deal w/ those as abstract as possible. (kicking ideas around as my brain warms up)